### PR TITLE
Update Fluent Bit container image

### PR DESCRIPTION
**Pull request by masa213f**
Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -5,8 +5,8 @@ const (
 	Version = "0.10.9"
 
 	// FluentBitImage is the image for slow-log sidecar container.
-	FluentBitImage = "quay.io/cybozu/fluent-bit:1.7.8.1"
+	FluentBitImage = "quay.io/cybozu/fluent-bit:1.8.11.1"
 
 	// ExporterImage is the image for mysqld_exporter sidecar container.
-	ExporterImage = "quay.io/cybozu/mysqld_exporter:0.13.0.1"
+	ExporterImage = "quay.io/cybozu/mysqld_exporter:0.13.0.3"
```
Comment (ymmt2005) [inline version.go:11]: I don't think `mysqld_exporter` needs an update.  It is a statically linked binary on a scratch image.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: version.go (hunk 1 above)
Comment (masa213f) [inline version.go:11]:
```suggestion
	ExporterImage = "quay.io/cybozu/mysqld_exporter:0.13.0.1"
```

Comment (masa213f) [reply version.go]: @ymmt2005 
Thank you. I reverted the exporter version.

**Review by ymmt2005 — APPROVED** [review]
LGTM.  But CI is failing due to the mysql-client package problem.

**Review by ymmt2005 — APPROVED** [review]

**Pull request merged**